client/cmd/dexc/config.go

```diff
@@ -149,6 +171,28 @@ func configure() (*Config, error) {
 	if cfg.Simnet && cfg.Testnet {
 		return nil, fmt.Errorf("simnet and testnet cannot both be specified")
 	}
+
+	// Calc RPC & Web addresses, if any of the hosts isn't available fallback
+	// to default network specific host
+	RPCHost := defaultHostByNetwork(cfg.RPCHost, cfg.Testnet, cfg.Simnet)
+	WebHost := defaultHostByNetwork(cfg.WebHost, cfg.Testnet, cfg.Simnet)
```

Comment:
We'll have cfg.Net set in the next switch below, so we can use that as an argument to `defaultHostByNetwork` if this is moved below that switch.

Reply:
True, nice observation